run_test.py

```diff
@@ -113,14 +113,16 @@
     elif args.test == 'chainer-doc':
         # See sphinx version RTD uses:
         # https://github.com/rtfd/readthedocs.org/blob/master/requirements/pip.txt
+        # Also note that NumPy 1.14 is required to run doctest, as doctests
```

Comment:
Strictly speaking `NumPy 1.14 or above` according to the implementation. A minor but do you think we should fix it?